### DISCUSSION TITLE
Change installation folder for Stride Engine?

### DISCUSSION THREAD

**Discussion by f1ow03** [stride3d/stride#2355] [Newcomers] [answered]
I installed the launcher on drive E: , but when I install the engine itself it installs on C: drive somewhere in AppData. I don't have enough space on C: and installation fails. Is there an option to install the engine on other drive?

**Comment by Doprez** [comment]
I believe the app installs using Nuget which is typically on the C drive. I don't think there is a way for the launcher to change this that I have tried. https://github.com/stride3d/stride/blob/dbb125e19a48715dd9fa7cf200a7d3f75f2fbb78/sources/launcher/Stride.Launcher/ViewModels/PackageVersionViewModel.cs#L20

So I think there "technically" should be a way to change it by manipulating nuget but i have not tested it and it would likely be a pain in the future based on other issues that can arise from changing nuget settings.

https://stackoverflow.com/questions/35710780/can-the-nuget-3-2-package-cache-location-be-changed This is likely what is needed but again I havent tested it so I cant promise anything.

**Reply by Doprez** [reply]
https://github.com/stride3d/stride/issues/10 seems like this has been an issue for a while sadly.

**Comment by Ethereal77** [comment, marked as answer]
While there's no solution for changing the installation directory (as Doprez has said, it depends on NuGet), I have succesfully moved my NuGet `cache` and `packages` folders out of my system SSD and into a second drive by setting a few environment variables:

![image](https://github.com/stride3d/stride/assets/8967302/e8441abc-f7e6-43c6-8cb3-db259a88f45a)

I hope this is useful.
